src/assets/data/edition/series/1/section/5/op25/source-description.json

```diff
@@ -845,7 +845,7 @@
                     "secondary": ["blauer Buntstift", "grüner Buntstift", "roter Buntstift", "blaue Tinte"]
                 },
                 "title": "<em>III. Satz op. 24</em> auf Bl. 2<sup>r</sup> System 9–10 links mit rotem Buntstift;<br /><em>„Sterne, Ihr silbernen Bienen - - -“</em> auf Bl. 5<sup>v</sup> oben links mit grünem Buntstift;<br /><em>„Des Herzens Purpurvogel“</em> auf Bl. 8<sup>v</sup> oben links mit rotem Buntstift.",
-                "date": "<em>22. VIII. 34</em> auf Bl. 2<sup>r</sup> oben links;<br /><em>19. IX. 34</em> auf Bl. 6<sup>r</sup> oben links;<br /><em>8. X. 34</em> auf Bl. 7<sup>v</sup> System 12–13 (zu <strong>M 321 Sk8</strong> T. 21B–26B);<br /><em>M. am 30. IX. nach Zürich abgereist.</em> auf Bl. 7<sup>v</sup> System 16 rechts;<br /><em>24. X</em>[.] <em>34</em> auf Bl. 8<sup>v</sup> System 4–5 halbrechts (zu <strong>M 322 Sk3</strong> T. {{ '{' }}1A{{ '}' }}, {{ '{' }}1B{{ '}' }}, {{ '{' }}1C‒2A{{ '}' }}, 1D‒2B);<br /><em>15. IX</em>[.] <em>34</em> auf Bl. 11<sup>r</sup> System 9–10 (nach SkII/10##?## T. 21);<br /><em>34 M. am</em> <em>9. XII.</em> <em>aus</em> <em>Z.</em> <em>zurückgekommen. | M. am</em> <em>10. I. 35</em> <em>operiert worden | Ma</em> [Unterführungszeichen für „am“] <em>21. I. </em>[Unterführungszeichen für „35“] [xx] auf Bl. 11<sup>r</sup> System 10–11 rechts.",
+                "date": "<em>22. VIII. 34</em> auf Bl. 2<sup>r</sup> oben links;<br /><em>19. IX. 34</em> auf Bl. 6<sup>r</sup> oben links;<br /><em>8. X. 34</em> auf Bl. 7<sup>v</sup> System 12–13 (zu <strong>M 321 Sk8</strong> T. 21B–26B);<br /><em>M. am 30. IX. nach Zürich abgereist.</em> auf Bl. 7<sup>v</sup> System 16 rechts;<br /><em>24. X</em>[.] <em>34</em> auf Bl. 8<sup>v</sup> System 4–5 halbrechts (zu <strong>M 322 Sk3</strong> T. {{ '{' }}1A{{ '}' }}, {{ '{' }}1B{{ '}' }}, {{ '{' }}1C‒2A{{ '}' }}, 1D‒2B);<br /><em>15. XI</em>[.] <em>34</em> auf Bl. 11<sup>r</sup> System 9–10 (nach <strong>M 322 Sk4</strong> T. 21C);<br /><em>34 M. am 9</em>[.] <em>XII. aus Z. zurückgeko<span style=\"text-decoration:overline;\">m</span>en. | M. am 10. I. 35 operiert worden | Ma</em> [Unterführungszeichen für „am“] <em>21. I.</em> [Unterführungszeichen für „35“] <em>erkrankt</em> auf Bl. 11<sup>r</sup> System 10–11 rechts.",
```

Comment:
Sind die Zeilenfallzeichen "|" im Original vorhanden, oder sollten diese ausserhalb der Kursivierung stehen?

Reply:
Ja, scheint auch uneinheitlich zu sein. Gibt mind. 25 Fälle mit nicht-kursivierten Zeilenfällen.

Und hier in der Datei gibt es auch noch dreimal das doppelte Zeichen "||". Intendiert?

Reply:
Uneinheitlich ist die Kursivierung, weil es bei der Erstellung in Word manchmal einfacher ist über den Zeilenfall hinweg zu markieren (und dann das Markierte zu kursivieren), manchmal aber auch nicht (wenn gerade ein "[xxx]" oder so dazwischen geschaltet ist). Sollen wir es systematisch durchgehen? Oder es zukünftig über das Converter-Script lösen? (`If "<em> ... | ... </em>" Then "<em> ...</em> | <em> ... </em>"`?) 

Das doppelte "||" kommt bei dem Jone-Gedichtext und soll 'größeren' Zeilenfall (sprich Strophenwechsel) andeuten. Nicht intuitiv klar?

Reply:
Einverstanden. Sollte besser systematisch gelöst werden.